data-prepper-plugins/trace-peer-forwarder-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/TracePeerForwarderProcessorConfig.java

```diff
@@ -0,0 +1,8 @@
+package org.opensearch.dataprepper.plugins.processor;
+
+import com.fasterxml.jackson.annotation.JsonClassDescription;
+
+@JsonClassDescription("The <code>trace_peer_forwarder</code> processor is used with peer forwarder to reduce by half " +
```

Comment:
nit:  `is used with peer forwarder to reduce the number of events by half that will be forwarded in a Trace Analytics pipeline.`